packages/client/src/request.d.ts

```diff
@@ -1,3 +1,3 @@
-import {OptionsWithUrl} from "request";
+import RequestOptions from "@sendgrid/helpers/classes/request";
 
-export type ClientRequest = OptionsWithUrl;
\ No newline at end of file
+export default RequestOptions;
```

Comment:
Can we just keep this a `ClientRequest` for compatibility (and less changes)?

Reply:
Done, also put back some other unnecessary changes